pcsx2/Patch_Memory.cpp

```diff
@@ -449,3 +466,18 @@ void _ApplyPatch(IniPatch *p)
 		break;
 	}
 }
+
+u64 SwapEndian(u64 InputNum, patch_data_type Type)
+{
+	if (Type == DOUBLE_LE_T) //DOUBLE_LE_T
+	{
+		InputNum = (InputNum & 0x00000000FFFFFFFF) << 32 | (InputNum & 0xFFFFFFFF00000000) >> 32; //Swaps 4 bytes
+	}
+	if ((Type == WORD_LE_T)||(Type==DOUBLE_LE_T)) //WORD_LE_T
+	{
+		InputNum = (InputNum & 0x0000FFFF0000FFFF) << 16 | (InputNum & 0xFFFF0000FFFF0000) >> 16; // Swaps 2 bytes
+	}
+	InputNum = (InputNum & 0x00FF00FF00FF00FF) << 8 | (InputNum & 0xFF00FF00FF00FF00) >> 8;   //Swaps 1 byte
+	return InputNum;
+}
```

Comment:
Please put spaces between // and the comments (like you did with // Swaps 2 bytes)

Reply:
Fixed in https://github.com/PCSX2/pcsx2/pull/4255/commits/006e946060582292f3e9b935f97268407d796010